services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/functionaltests/BasicCrudTest.java

```diff
@@ -360,10 +371,17 @@ public void putWithConditionThatFails() {
                                                   .conditionExpression(conditionExpression).build());
     }
 
+    @Test
+    public void deleteItemInterfaceToOperationSucceeds() {
+        mappedTable.deleteItem(r -> r.key(Key.create(stringValue("id-value"), stringValue("sort-value"))));
+        mappedTable.deleteItem(DeleteItemEnhancedRequest.builder()
+                                                        .key(Key.create(stringValue("id-value"), stringValue("sort-value")))
+                                                        .build());
+    }
+
```

Comment:
I think we should be able to make basic assertions here beyond 'it didn't blow up'.